corehq/motech/fhir/models.py

```diff
@@ -298,3 +303,29 @@ def get_resource_type_or_none(case, fhir_version) -> Optional[FHIRResourceType]:
         )
     except FHIRResourceType.DoesNotExist:
         return None
+
+
+class FHIRImporter(models.Model):
+    domain = models.CharField(max_length=127, db_index=True)
+    connection_settings = models.ForeignKey(
+        ConnectionSettings,
```

Comment:
I gotta say, this looks like it's turned out to be a really helpful abstraction.